plugins/generator-1.17.1/forge-1.17.1/utils/mcelements.ftl

```diff
@@ -0,0 +1,7 @@
+<#function toResourceLocation string>
+    <#if string?matches('"[^+]*"')>

```

Comment:
This regex will also match this:

![image](https://user-images.githubusercontent.com/16374228/132381154-a39d9e40-c3ba-4e44-bf8a-8aaff288d62f.png)

Here, tolowercase should be called, but in your case, will not be

Reply:
The operator returns true only if the entire string matches the regex, so it does add `.toLowerCase`

Reply:
I see